src/main/java/uk/gov/hmcts/reform/sendletter/tasks/FailedToPrintTask.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.sendletter.tasks;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
+import uk.gov.hmcts.reform.sendletter.entity.LetterState;
+import uk.gov.hmcts.reform.sendletter.logging.AppInsights;
+
+import java.sql.Timestamp;
+import java.time.LocalTime;
+import java.time.ZonedDateTime;
+
+public class FailedToPrintTask {
+    private static final LocalTime STALE_CUT_OFF_TIME = LocalTime.of(17, 0, 0);
```

Comment:
latest cutoff time is 16.00

Reply:
true. wrote this code before the altered time 😉